ci/docker/python-wheel-windows-vs2017.dockerfile

```diff
@@ -83,7 +83,7 @@ RUN choco install -r -y --no-progress python --version=%PYTHON_VERSION%
 RUN python -m pip install -U pip setuptools
 
 COPY python/requirements-wheel-build.txt arrow/python/
-RUN python -m pip install -r arrow/python/requirements-wheel-build.txt
+RUN python -m pip install tzdata -r arrow/python/requirements-wheel-build.txt
```

Comment:
I think the tzdata should be installed in the env for testing the wheel, and shouldn't be required when _building_. 
We do have a `python_wheel_windows_test.bat` script which I suppose is relevant here.

Reply:
And checking the log output, you can indeed already see that the above did not work to install tzdata in the test, because the skip is correctly working: https://github.com/ursacomputing/crossbow/runs/6089209704?check_suite_focus=true#step:7:441

Reply:
Great, thanks for the pointers! I wasn't sure where to put it =)